notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.        TODO: independent batched version of this function
</pre><p>it is your TODO or model authors? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>